### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: fuzz/src/fuzz.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import org.iban4j.Iban;
```
Comment (xansec) [inline fuzz/src/fuzz.java:1]: Can we rename this to reflect what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: fuzz/src/fuzz.java (hunk 1 above)
Comment (ebell495) [reply fuzz/src/fuzz.java]: I've renamed it

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**